src/App.js

```diff
@@ -1,18 +1,34 @@
 import './App.css';
-import initialCards from './cards-data';
+// import initialCards from './cards-data';
 import Player from './components/Player';
 import CardList from './components/CardList';
-import { useState } from 'react';
+// import { useState } from 'react';
 import ExecutePassButton from './components/ExecutePassButton';
+import { useGameContext } from './context/GameContext';
 
 function App() {
-  const [deck, setDeck] = useState(initialCards);
-  const [playerOneHand, setPlayerOneHand] = useState([]);
-  const [selectedCard, setSelectedCard] = useState();
-  const [playerTwoHand, setPlayerTwoHand] = useState([]);
-  const [playerThreeHand, setPlayerThreeHand] = useState([]);
-  const [from, setFrom] = useState('deck');
-  const [to, setTo] = useState(1);
+  // const [deck, setDeck] = useState(initialCards);
+  // const [playerOneHand, setPlayerOneHand] = useState([]);
+  // const [selectedCard, setSelectedCard] = useState();
+  // const [playerTwoHand, setPlayerTwoHand] = useState([]);
+  // const [playerThreeHand, setPlayerThreeHand] = useState([]);
+  // const [from, setFrom] = useState('deck');
+  // const [to, setTo] = useState(1);
+
+  const {
+    playerOneHand,
+    setPlayerOneHand,
+    playerTwoHand,
+    setPlayerTwoHand,
+    playerThreeHand,
+    setPlayerThreeHand,
+    to,
+    from,
+    deck,
+    setDeck,
+    selectedCard,
+    setSelectedCard,
+  } = useGameContext();
```

Comment:
is ```App.js``` using all of these? i don't think so (could be wrong!). if not, you only want to import from context those files which are needed in that component